README.md

```diff
@@ -1,8 +0,0 @@
-Zenoss Zenpack for monitoring MySQL server
```

Comment:
I can't install the ZenPack because `README.md` was deleted, and `setup.py` depends on it.

```
(zenoss) [zenoss@2e30506c9021 ~]$ zenpack --link --install /mnt/src/ZenPacks.zenoss.MySqlMonitor
Traceback (most recent call last):
  File "setup.py", line 29, in <module>
    dirname(__file__) + 'README.md').read().split('\n'))[0],
IOError: [Errno 2] No such file or directory: 'README.md'
```